src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -79,12 +79,25 @@ public sealed class ContentManager : DisposableBase
 
             IniDataContext = new IniDataContext(fileSystem);
 
-            IniDataContext.LoadIniFiles(@"Data\INI\Default\Object.ini");
-            IniDataContext.LoadIniFiles(@"Data\INI\Object");
-            IniDataContext.LoadIniFiles(@"Data\INI\multiplayer.ini");
-            IniDataContext.LoadIniFile(@"Data\INI\PlayerTemplate.ini");
-
-            IniDataContext.LoadIniFile(@"maps\MapCache.ini");
+            // TODO: Add these into IGameDefinition? Should we preload all ini files?
+            switch (sageGame)
+            {
+                case SageGame.Ra3:
+                case SageGame.Ra3Uprising:
+                case SageGame.Cnc4:
+                    // TODO
+                    break;
+                default:
```

Comment:
I don't think you have captured all the (existing) game-specific details around which .ini files get loaded. For example, `ParticleSystem.ini` was previously loaded for `Bfme2Rotwk` and below. `GameData.ini` and `Mouse.ini` were loaded for `Cnc3KanesWrath` and below.